packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/iam/disable/iam-disable.controller.js

```diff
@@ -0,0 +1,59 @@
+import { TRACKING_TASK_TAG } from './iam-disable.constant';
+
+export default class IamDisableController {
+  /* @ngInject */
+  constructor($translate, DedicatedCloud) {
+    this.$translate = $translate;
+    this.DedicatedCloud = DedicatedCloud;
+    this.boundOnCloseClick = this.onCloseClick.bind(this);
+  }
+
+  $onInit() {
+    this.TRACKING_TASK_TAG = TRACKING_TASK_TAG;
+
+    this.task = null;
+
+    this.loaders = {
+      action: false,
+      init: true,
+    };
+
+    return this.DedicatedCloud.isIamTogglable(this.productId, 'enabled', true)
+      .then((response) => {
+        this.iamTogglable = response.toggable;
+      })
```

Comment:
```suggestion
    return this.DedicatedCloud.isIamTogglable(this.productId, 'enabled')
      .then((isIamTogglable) => {
        this.iamTogglable = isIamTogglable;
      })
```
https://github.com/ovh/manager/pull/9649#discussion_r1277304830